### DISCUSSION TITLE
Shlink not available

### DISCUSSION THREAD

**Discussion by manuelkamp** [shlinkio/shlink#1575] [Help wanted] [answered]
Hello, I installed Shlink (via Installer Script) to be served with Roadrunner, but only CLI works (I can generate Shortlinks via CLI). But I cannot access Shlink via Browser (also not via curl on localhost). I do not know what is wrong, I cannnot find any logs (/opt/shlink/data/log is empty).

* Shlink Version: 3.3.0
* PHP Version: 8.1
* How do you serve Shlink: Self-hosted RoadRunner
* Database engine used: Dedicated MySQL Server

#### Summary

* Using ./cli short-url:create works and I can see the correct one in ./cli short-url:list
* curl localhost:8080 -i gives: Failed to connect to localhost port 8080: Connection refused
* browsing on any client and any browser in local network to the server on port 8080 gives something like: ERR_CONNECTION_REFUSED
* I also configured my reverse proxy (nginx), because I wanted to try access via ssl from the internet - also the Connection Refused error.

**Comment by mikepayne02** [comment]
Did you ever find out why this error was popping up?

**Comment by manuelkamp** [comment, marked as answer]
No, I wiped the LXC and reinstalled everything and now it works

**Reply by mikepayne02** [reply]
That’s interesting, because I tried running the container from scratch in a Docker playground and got the same error with the base image. Using the roadrunner image fixes it but I think for now I’ll just deal with it because the request still completes.
